Readme.md

```diff
@@ -15,24 +15,52 @@ TL;DR Dive into coding right away by clicking here:
 We are an open community of volunteers without a commercial purpose. We believe that through a utilitarian approach, we can do the most good in the quickest time. Applying unused engineering we can help the world cope with the threat of COVID-19.
 
 #### Python Version
+
 covidX will be run on python *3.7.6* and *3.8.5*
 
 #### Required Services for Self-Hosting
+
 The following steps are necessary in order to ensure that you are able to self-host your own instance of the project smoothly:
 * Install [PostgreSQL](https://www.postgresql.org/download/)
 * Install [Bazel](https://docs.bazel.build/versions/master/install.html)
 * Sign up to [Docker Hub](https://hub.docker.com/signup) and install [Docker Desktop](https://www.docker.com/get-started)
 * Sign up to [Auth0](https://auth0.com/) (obtain domain, client id, and client secret)
 * Sign up to [Algolia](https://www.algolia.com/users/sign_up) (obtain application id and admin API key)
 
-#### How to Run Locally Using Docker
+#### How to Run Locally Using Docker (docker-compose)
+
+* Create your `.env` file from the sample:
+
+```shell
+cp .env_sample .env
+```
+
+* Fill in the details based on the Auth0 and Algolia sign-ups
+
+See https://github.com/Xcov19/covidX/issues/78
+
+* Generate a private / public key pair
+
 ```shell
-docker run --rm -it -e SECRET_KEY=<YOUR_DJANGO_SECRET_KEY> -e DEBUG_ENV=1 -e CPPFLAGS="$(pg_config --cppflags)" -e LDFLAGS="$(pg_config --ldflags)" -e SECRET_ID="SECRET_KEY" -e BUCKET_NAME="gae-bizlead" -e DJANGO_SETTINGS_MODULE="covidX.settings" -e WSGI_APPLICATION="covidX.wsgi.application" --security-opt apparmor=unconfined codecakes/covidx_manage:bazel runserver_plus
+openssl req -x509 -newkey rsa:4096 -keyout privateKey.key -out certificate -days 365 -nodes
 ```
 
-For local development `YOUR_DJANGO_SECRET_KEY` can be an alphanumeric string of your choosing.
+* Build and start the app
+
+```shell
+docker-compose up
+```
+
+The app should be built automatically on first run.
+
+Note: to reflect changes made to `.env` after the application has been brought `up`, you need to force a rebuild:
+
+```shell
+docker-compose --build
+```
```

Comment:
@codecakes Is this [previous section](https://github.com/marcelloromani/covidX/blame/docs/update_local_run_readme/Readme.md#L109-L124) needed now that we can simply do `docker-compose up`?

Reply:
docker build has nothing to do with previous section. ive already told to rename previous section for developers. "If you don't know bazel, dont bothe with this section" @marcelloromani @milan-tom

Reply:
Added that line, hopefully in the right place.